Makefile

```diff
@@ -64,9 +64,9 @@ test-resetdb: node_modules
 	docker rm --force --volumes hydra_test_database_mysql || true
```

Comment:
Please revert the changes in these files because this would need a change in other files (CI, dockertest) as well to keep consistency, and it's unrleated to this PR. Thanks!